modules/core/arrow-core-data/src/test/kotlin/arrow/core/SortedMapKTest.kt

```diff
@@ -32,6 +45,54 @@ class SortedMapKTest : UnitSpec() {
         SortedMapK.traverse<String>(),
         SortedMapK.functor<String>(),
         { a: Int -> sortedMapOf("key" to a).k() },
-        EQ))
+        EQ),
+      SemialignLaws.foldablelaws(SortedMapK.semialign<String>(),
```

Comment:
Why `foldablelaws` instead of `laws`?

Reply:
Semialign has additional laws for functors that are also foldable. For reference: https://hackage.haskell.org/package/semialign-1.1/docs/Data-Semialign.html

Reply:
But seeing this I'd prefer an overload instead of a seperate name o.O

Reply:
> Semialign has additional laws for functors that are also foldable

Got it 👍 It can be renamed to `laws` as well since it has a different signature.

Reply:
renamed it